float32 thrust
float32[4] q
float32[3] ang_vel
float32[3] ang_acc
